v2/client.go

```diff
@@ -32,8 +32,9 @@ type (
 		// Tailnet allows specifying a specific Tailnet by name, to which this Client will connect by default.
 		Tailnet string
 
-		// http is the http client to use for requests to the API server. If specified, this supercedes the above configuration.
-		http *http.Client
+		// HTTP is the [http.Client] to use for requests to the API server.
+		// If not specified, a new [http.Client] with a Timeout of 1 minute will be used.
```

Comment:
Potentially worth flagging the caveat around the client being replaced when `UseOAuth` is called at this level as well.